fauna/manifest.json

```diff
@@ -0,0 +1,39 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "329e2845-de4a-4477-b5e9-c85f7c707316",
+  "app_id": "fauna",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "uninstallation": "README.md#Uninstallation",
+    "changelog": "CHANGELOG.md",
+    "description": "Import your Fauna query logs into Datadog.",
+    "title": "Fauna",
+    "media": [],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS",
+      "Category::Cloud",
+      "Category::Data Stores",
+      "Category::Log Collection"
```

Comment:
This should include Submitted Data Type and Offering. Please see here https://docs.datadoghq.com/developers/integrations/check_references/#classifier-tags